### PR TITLE
Exposes top-level attributes of the codebuild project

### PR DESCRIPTION
**Pull request by lorengordon**
Original goal was to support setting the `build_timeout`, since terrafirm timeouts are longer than the default 60min and if the codebuild job times out then it's manual cleanup to terminate the resources. But, figured easy enough to add the rest of the simple top-level attributes for the codebuild project...

### CODE REVIEW

**Comment by lorengordon** [comment]
@plus3it/terraform This pr is ready for review now

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,10 @@ __pycache__
 .env
 .private
 *.tfvars
+
+# tardigrade-ci
+.tardigrade-ci
```
Comment (eemperor) [inline .gitignore:16]: You're ignoring and committing the file?  Or maybe commit by mistake?

File: modules/tag/variables.tf [hunk 2]
```diff
@@ -9,6 +9,46 @@ variable "buildspec" {
   default     = ""
 }
 
+variable "buildspec" {
+  type        = string
+  description = "Buildspec used when a tag reference is created or updated"
+  default     = ""
+}
```
Comment (eemperor) [inline modules/tag/variables.tf:16]: This is a duplicate I believe

Comment (lorengordon) [reply modules/tag/variables.tf]: fixed

File: .gitignore (hunk 1 above)
Comment (lorengordon) [reply .gitignore:16]: oops, wasn't supposed to commit that, it's pulled in automatically by the make target

**Comment by lorengordon** [comment]
@eemperor great catches, thanks! all fixed up

**Pull request merged**